packit/upstream.py

```diff
@@ -286,19 +286,20 @@ def create_patches(
             patch_msg = f"{commit.summary}\nAuthor: {commit.author.name} <{commit.author.email}>"
 
             logger.debug(f"PATCH: {patch_name}\n{patch_msg}")
+            git_diff_cmd = [
+                "git",
+                "diff",
+                "--patch",
+                parent.hexsha,
+                commit.hexsha,
+                "--",
+                ".",
+            ] + [
+                f'":(exclude){sync_file.src}"'
```

Comment:
I'd say this is a good start but in the end we should have a dedicated config option for this. Let's give this a try and we shall see.